setup.py

```diff
@@ -15,10 +17,41 @@ def get_version():
             ver_file.close()
 
 
+def get_data_files():
+
+    def get_completion_install_location(shell):
+        uname = platform.uname()[0]
+        is_root = (os.geteuid() == 0)
+        prefix = ''
+        if shell == 'bash':
+            if is_root and uname == 'Linux':
+                prefix = '/'
+            location = os.path.join(prefix, 'etc/bash_completion.d')
+        elif shell == 'zsh':
+            if is_root and uname == 'Darwin':
```

Comment:
if is_root, the bash/zsh completion files will be installed at `/usr/share/zsh/site-functions`, but if not, installed at `sys.prefix` + `share/zsh/site-functions`. so it will be like:

* /usr/local/share/zsh/site-functions
* /Users/ken/venv/share/zsh/site-functions

when sys.prefix is `/System/Library/Frameworks/Python.framework/Versions/2.7`, the user should be root user,
so the completion files will be installed at `/usr/share/zsh/site-functions`. And I think it is correct place.